client/components/home/Pickup/carousel.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Flex, Stack, HStack, Image } from "@chakra-ui/react";
+import { useState, useEffect } from "react";
+import { COLORS } from "theme";
+import { useBreakpointValue } from "@chakra-ui/react";
+
+const PickupCarousel = () => {
+	const slides = [
+		{
+			url: useBreakpointValue({
+				base: "/headerBanner1mobile.png",
+				md: "/headerBanner1desktop.png",
+			}),
+			number: 20,
+		},
+		{
+			url: useBreakpointValue({
+				base: "/headerBanner2mobile.png",
+				md: "/headerBanner2desktop.png",
+			}),
+			distance: "20m",
+		},
+	];
+	const [currentSlide, setCurrentSlide] = useState(0);
+	const slidesCount = slides.length;
+	const SLIDES_INTERVAL_TIME = 20000;
```

Comment:
if SLIDES_INTERVAL_TIME and ANIMATION_DIRECTION won't be changed, can consider initialising them outside of the component

Reply:
moved SLIDES_INTERVAL_TIME out of the component, removed ANIMATION_DIRECTION as per previous comment.